### PR TITLE
Convert the BIM tileset to avoid using techniques

### PR DESCRIPTION
**Pull request by ptrgags**
Since we deprecated the `KHR_techniques_webgl` extension, we wanted to update the BIM tileset used in several Sandcastle examples so they don't break when we remove the old Model (See #10379 and #10346).

The tileset was non-trivial to convert, and visually there are some differences including:
* The materials now use PBR, so the lighting looks a bit different
* Some dashed polylines in the original model now render as solid, as the custom GLSL code to render them doesn't translate directly to GLSL.

This PR updates the relevant sandcastles to use the new asset ID. It also adjusts the timeline and other minor visual tweaks.

Also note that I only updated thumbnails where the difference in lighting would be noticeable. 

Note that if you set `ExperimentalFeatures.enableModelExperimental = true`, all 5 of the affected sandcastles will render without crashing.

@j9liu could you review?

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @ptrgags!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/CesiumGS/cesium/blob/bim-no-techniques/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/CesiumGS/cesium/tree/main/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.

**Comment by lilleyse** [comment]
Could you resize the new image to 225x150?

image

File: Apps/Sandcastle/gallery/Polylines on 3D Tiles.html [hunk 1]
```diff
@@ -39,12 +39,56 @@
...
         const powerplant = scene.primitives.add(
           new Cesium.Cesium3DTileset({
-            url: Cesium.IonResource.fromAssetId(8564),
+            url: Cesium.IonResource.fromAssetId(1240402),
             show: false,
           })
         );
+
+        // Hide duplicate floor geometry to avoid
+        // z-fighting artifacts.
```
Comment (lilleyse) [inline Apps/Sandcastle/gallery/Polylines on 3D Tiles.html:54]: Glad to see this artifact finally addressed!

**Comment by ptrgags** [comment]
@lilleyse I updated the images and squashed the commits!

**Pull request merged**